api/swagger.yaml

```diff
@@ -2725,6 +2725,11 @@ definitions:
             description: "Run an init inside the container that forwards signals and reaps processes. This field is omitted if empty, and the default (as configured on the daemon) is used."
             type: "boolean"
             x-nullable: true
+          Sysctls:
+            description: "Set kernel namedspaced parameters (sysctls) in the container."
+            type: "object"
+            additionalProperties:
+              type: "string"
```

Comment:
Can you add an example here?

```yaml
Sysctls:
  description: "Set kernel namedspaced parameters (sysctls) in the container."
  type: "object"
  additionalProperties:
    type: "string"
  example:
    net.ipv4.tcp_keepalive_time: "600"
    net.ipv4.tcp_keepalive_intvl: "60"
    net.ipv4.tcp_keepalive_probes: "3"
    net.ipv4.tcp_timestamps: "0"
```

Reply:
Is there a way to reference or crosslink to the documentation for this option on the container `HostConfig`?

Reply:
Hm; good one, perhaps just refer to it in the description 🤔 ("this option accepts the same sysctls as can be specified for containers", or something along those lines 😂)